superset/migrations/versions/bc649e169f14_support_for_lookup_extraction_filters_.py

```diff
@@ -0,0 +1,670 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""support for lookup extraction filters and queries
+
+Revision ID: bc649e169f14
+Revises: c82ee8a39623
+Create Date: 2019-03-14 14:51:02.124189
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = 'bc649e169f14'
+down_revision = 'c82ee8a39623'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_table('long_lat')
```

Comment:
this migration doesn't look right, it's doing unrelated operations

Reply:
yeah sorry my bad. This migration is not needed at all! I removed it